.github/workflows/lint-test.yaml

```diff
@@ -216,12 +203,6 @@ jobs:
     if: contains(github.event.pull_request.labels.*.name, 'CICD:build-consensus-only-image')
     steps:
       - uses: actions/checkout@v4
-      - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
-        with:
-          GIT_CREDENTIALS: ${{ secrets.GIT_CREDENTIALS }}
```

Comment:
Why did we have this before? Do we install something that needs the credentials?

Reply:
I'm not really sure. I think that cargo might need to use git to fetch a few dependencies. We probably don't need this anymore since the deps are now included in the runner image